huaweicloud/services/acceptance/apig/resource_huaweicloud_apig_group_test.go

```diff
@@ -46,40 +52,102 @@ func TestAccGroup_basic(t *testing.T) {
 		CheckDestroy:      rc.CheckResourceDestroy(),
```

Comment:
The current merged test case will cause the `checkDestroy` of other resources to not be verified.
It is recommended to write a public function to verify whether all resources are successfully deleted.

Reply:
done.